docs/reference/src/introduction/index.md

```diff
@@ -2,22 +2,24 @@
 
 # Getting Started
 
-First, you need `.slint` files, which contain a textual description of your user interface, written in the Slint language.
-Next, you embed the `.slint` files in a project written in a supported programming language, like C++, Rust, or JavaScript.
+Using Slint in your projects needs two components:
 
-There are three different pathways to get started with Slint:
+1. `.slint` files, which contain a textual description of the user interface, written in the Slint language.
+2. The programming language of the project where you embed the `.slint` files.
+
+    - Slint supports C++, Rust, and JavaScript.
 
-1. [SlintPad](https://slint.dev/editor) - Use this to get a feel of the Slint language.
-   This is a web browser-based tool where you can try Slint out.
+There are three different pathways to get started with Slint:
 
-2. As a UI Designer, working with Slint files locally, we recommend the following combination of software tools:
+1. [SlintPad](https://slint.dev/editor) - Use this browser-based tool to get a feel for the Slint language.
+2. As a **UI Designer**, working with Slint files locally, we recommend the following combination of software tools:
 
-   - [Visual Studio Code](https://code.visualstudio.com)
-   - [The Slint for Visual Studio Code Extension](https://marketplace.visualstudio.com/items?itemName=Slint.slint)
+    - [Visual Studio Code](https://code.visualstudio.com)
+    - [The Slint for Visual Studio Code Extension](https://marketplace.visualstudio.com/items?itemName=Slint.slint)
 
-3. As a Software Developer, integrating Slint into a new or existing code base, choose one of these languages to
+3. As a **Software Developer**, integrating Slint into a new or existing code base, choose one of these languages to
    get started:
 
-   - [C++](slint-cpp:index.html)
-   - [Rust](slint-rust:index.html)
-   - [JavaScript](slint-node:index.html)
+    - [C++](slint-qs:index.html)
```

Comment:
I added this in the QS PR, I think it makes more sense now? But the intro page also now has no language-specific content…

Reply:
I thought I saw it, but now I don't see it anymore. I think it's missing?

Reply:
@tronical I'll take a look. But should it still be this general format? The `saint-lang` definition taken from the config file for each area?

My more general change here was to change the "getting started" link to the actual quickstart. Do you think that makes sense?

Reply:
Ah sorry, I misunderstood. Difficult call. It's good to have links within the reference, instead of links away from it so early on. I think it makes sense, but it would be really good to also have links to the C++/Rust/JS reference docs as well somewhere on this page.

If the links here remain language agnostic, then perhaps it's enough to just refer to the "Quick Start" regardless of the language. But I think it would be even better of those links here would get the reader to the quick start with the correct tabs already being active. Is that possible?

Reply:
@tronical It's certainly possible, and I took a good look, but ran into difficulty trying to maintain "state" between the links and tabs in the quickstart section. I think this is something I need to resolve in the medium term, but for now, I have added links to both.